crates/rpc/src/handlers/state.rs

```diff
@@ -20,3 +45,9 @@ pub async fn get_state_from_id(state_id: ID, db: &ReamDB) -> Result<BeaconState,
             "Failed to find `beacon_state` from {block_root:?}"
         )))
 }
+
+pub async fn get_state(state_id: ID, db: ReamDB) -> Result<impl Reply, Rejection> {
+    let state = get_state_from_id(state_id, &db).await?;
+
+    Ok(with_status(BeaconResponse::json(state), StatusCode::OK))
+}
```

Comment:
```suggestion
}

```

missing new line 
![image](https://github.com/user-attachments/assets/85d3f614-cfcf-4e9c-8738-33b7a2f2d728)